src/store/getters.ts

```diff
@@ -1233,6 +1256,6 @@ export const isMrasRequestType = (state: StateIF): boolean => {
 
 /** True if jurisdiction is a MRAS jurisdiction. */
 export const isMrasJurisdiction = (state: StateIF): boolean => {
-  const { xproJurisdiction } = getNrData(state)
+  const xproJurisdiction = getJurisdictionText(state)
```

Comment:
The old code depended on `startAnalyzeName` action being called, but:
- we need to identify if this is a MRAS jurisdiction before that
- we really don't need to look at the NR data for this (this is the object that will be posted to the API)
- we can just get the xpro jurisdiction text from another getter which, itself, doesn't depend on anything we don't already have